### PR TITLE
History Page

### PR DESCRIPTION
**Pull request by anshikaa12**
>>created history Page
>>by clicking on the video card you can add it to the history

### CODE REVIEW

**Review by ArjunGTX — APPROVED** [review]
everything looks good

File: src/pages/history/history.jsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            >
+              CLEAR HISTORY
+            </button>
+          </div>
+          <div className="history-sec-list">
+            {historyState.historyList.map((vid) => {
+              return (
+                <div className="basic-card history-basic-card">
+                  <div className="card-header">
+                    <img
```
Comment (munjalSaksham71) [inline src/pages/history/history.jsx:35]: maybe add ```alt``` attribute for image?

**Review by munjalSaksham71 — APPROVED** [review]
Code is understandable and clean
Folder structure is good too>

Just suggested one thing below- please Refer.

File: src/pages/history/history.jsx (hunk 1 above)
Comment (anshikaa12) [reply src/pages/history/history.jsx:35]: already added!

**Pull request merged**